now-build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
+
+cd ../../
+export NODE_ENV="production"
+yarn concurrently 'yarn workspace @magento/pwa-buildpack run build' 'yarn workspace @magento/peregrine run build:esm'
```

Comment:
Why not just `yarn run build`? Would essentially do all of the things on line 5 and 6 with some extra steps. Was it the extra steps we're trying to avoid here?

Reply:
Yeah, this was to speed up deployments. In deployments we don't need to build Venia's ES modules for distribution, and we don't need to validate queries. It's a speedup of almost half a minute!

Reply:
Oh my!